### ISSUE TITLE
Change default folder for save_obs

### ISSUE THREAD

**Issue by justinkterry** [PettingZoo-Team/PettingZoo#297]
This code snippet:
```
import supersuit as ss
from pettingzoo.butterfly import pistonball_v3
from pettingzoo.utils import save_observation

env = pistonball_v3.env(n_pistons=6, local_ratio=0.2, time_penalty=-0.1, continuous=True, random_drop=True, random_rotate=True, ball_mass=0.75, ball_friction=0.3, ball_elasticity=1.5, max_cycles=900)

env = ss.color_reduction_v0(env, mode='B')
#env = ss.dtype_v0(env,'float32')
env = ss.resize_v0(env, x_size=20, y_size=76)
# env = ss.frame_stack_v1(env,1)

env.reset()
save_observation(env, all_agents=True)
```

Saves the observations in a folder in the working directory labeled "supersuit.aec_wrappers." Surely we can do better than this.

**Comment by weepingwillowben** [comment]
This seems like a good use case for a __str__ method in the wrappers.

Gym does a really good job of this with just 4 lines of code

https://github.com/openai/gym/blob/master/gym/core.py#L251

I didn't quite realize how simple it was until I looked

**Comment by justinkterry** [comment]
Closed by #302
